### ISSUE TITLE
result ordering

### ISSUE THREAD

**Issue by missinglink** [pelias/api#150]
this one I can't explain, @hkrishna any ideas?

http://pelias.stage.mapzen.com/search?size=3&input=soho,%20new%20york

when searching for 'soho, new york' for some unknown reason `sohotel` comes higher than `SoHo`:
```javascript
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "properties": {
        "id": "24068:_:_:_:soho",
        "layer": "neighborhood",
        "name": "Soho",
        "alpha3": "USA",
        "admin0": "United States",
        "admin1": "New York",
        "admin1_abbr": "NY",
        "admin2": "New York County",
        "text": "Soho, New York County, NY"
      },
      "geometry": {
        "type": "Point",
        "coordinates": [
          -74.007362241776,
          40.725167878971
        ]
      }
    },
    {
      "type": "Feature",
      "properties": {
        "id": "6479010",
        "layer": "geoname",
        "name": "sohotel",
        "alpha3": "USA",
        "admin0": "United States",
        "admin1": "New York",
        "admin1_abbr": "NY",
        "admin2": "New York County",
        "local_admin": "Manhattan",
        "locality": "New York",
        "neighborhood": "Bowery",
        "category": [
          "accommodation"
        ],
        "address": {
          
        },
        "text": "sohotel, Manhattan, NY"
      },
      "geometry": {
        "type": "Point",
        "coordinates": [
          -73.99449,
          40.71952
        ]
      }
    },
    {
      "type": "Feature",
      "properties": {
        "id": "5138691",
        "layer": "geoname",
        "name": "SoHo",
        "alpha3": "USA",
        "admin0": "United States",
        "admin1": "New York",
        "admin1_abbr": "NY",
        "admin2": "New York County",
        "local_admin": "Manhattan",
        "locality": "New York",
        "category": [
          "admin"
        ],
        "address": {
          
        },
        "text": "SoHo, Manhattan, NY"
      },
      "geometry": {
        "type": "Point",
        "coordinates": [
          -73.99961,
          40.72241
        ]
      }
    }
  ],
  "bbox": [
    -74.007362241776,
    40.71952,
    -73.99449,
    40.725167878971
  ],
  "date": 1435827175906
}
```

**Comment by missinglink** [comment]
possibly related? http://pelias.compare.s3-website-eu-west-1.amazonaws.com/#/search%3Flayers=admin2&input=bronx

**Comment by missinglink** [comment]
uh-oh, so.. I figured this one out and it's a real mind-bender...

I imported `quattro:admin2` in to my local and ran the above query for the bronx, and it came back as I was expecting with `Bronx County` as the top result, confusing...

so I decided to up the shard count to 24 (to match stage) and hey presto! the results are the same ordering as the staging server.

it appears that the per-shard TF/IDF is messing with the sorting order.
